docs/source/noise/index.rst

```diff
@@ -130,7 +130,7 @@ Noise types for each column
     - Leave a field blank, use a fake name, make phonetic errors, make OCR errors, make typos
     - Last names use a different list of fake names than the list for first names. In the 1040 form, the same noise types apply to the last name columns for the joint filer and dependents
   * - Age
-    - Decennial Census, ACS, CPS, W-2 and 1099
+    - Decennial Census, ACS, CPS
```

Comment:
This is a separate issue that I just noticed: WIC is not listed here, which is correct... but on the Datasets page, 'Age' is listed as a column of WIC, which is incorrect.

We should remove age from the WIC data as a separate hot fix because the documentation doesn't match what's in the released data.

Reply:
Ah, thanks for catching. I will make a Jira ticket now to make this hot-fix!